source/common/network/socket_option_impl.h

```diff
@@ -53,6 +53,12 @@ namespace Network {
 #define ENVOY_SOCKET_SO_REUSEPORT Network::SocketOptionName()
 #endif
 
+#ifdef UDP_GRO
+#define ENVOY_SOCKET_UDP_GRO ENVOY_MAKE_SOCKET_OPTION_NAME(SOL_UDP, UDP_GRO)
+#else
+#define ENVOY_SOCKET_UDP_GRO ENVOY_MAKE_SOCKET_OPTION_NAME(SOL_UDP, GRO_UDP)
+#endif
```

Comment:
How about follow other macros in this file, make ENVOY_SOCKET_UDP_GRO an empty SocketOptionName if UDP_GRO is not defined? 

```
#ifdef UDP_GRO
#define ENVOY_SOCKET_UDP_GRO ENVOY_MAKE_SOCKET_OPTION_NAME(SOL_UDP, UDP_GRO)
#else
#define ENVOY_SOCKET_UDP_GRO Network::SocketOptionName()
#endif
```

Reply:
This change was fixed with the previous update! Thanks for pointing this out!